.github/workflows/ci.yml

```diff
@@ -170,27 +170,18 @@ jobs:
           key: cargo-${{ matrix.config.target }}-${{ matrix.config.variant }}-${{ matrix.config.v8_enable_pointer_compression }}-${{ hashFiles('Cargo.lock', 'build.rs', 'git_submodule_status.txt') }}
           restore-keys: cargo-${{ matrix.config.target }}-${{ matrix.config.variant }}-${{ matrix.config.v8_enable_pointer_compression }}-
 
-      - name: Install and start sccache
-        shell: pwsh
+      - name: Install sccache
+        uses: mozilla-actions/sccache-action@v0.0.8
+
+      - name: Start sccache
         env:
+          # Start sccache server instead of using `RUSTC_WRAPPER` environment variable (recommended by `sccache-action`).
+          # Because it cannot compile `syn` lib correctly on windows.
           SCCACHE_DIR: ${{ github.workspace }}/target/sccache
-          SCCACHE_CACHE_SIZE: 128M
+          SCCACHE_CACHE_SIZE: 1G
```

Comment:
Set larger `1G` to improve cache hit rate.

Reply:
I wonder if set `SCCACHE_GHA_ENABLE: "true"` here, if it could improves the cache hit rate much better?